### PR TITLE
Implement ENTRIES function

### LINKED ISSUE #1290: Implementation ENTRIES for MAP
# Description
Implementation ENTRIES()

# Example
Function ENTRIES()

```
SELECT ENTRIES(Bar)  FROM Foo;
```

# Task

# Reference

### PR DESCRIPTION
**Pull request by 2-NOW**
resolve #1290

### CODE REVIEW

File: core/src/executor/evaluate/function.rs [hunk 1]
```diff
@@ -763,3 +763,16 @@ pub fn length<'a>(name: String, expr: Evaluated<'_>) -> Result<Evaluated<'a>> {
...
+    match expr.try_into()? {
+        Value::Map(expr) => {
+            let entries = expr
+                .into_iter()
+                .map(|(k, v)| Value::List(vec![Value::Str(k), v]))
+                .collect::<Vec<_>>();
+            Ok(Evaluated::Value(Value::List(entries)))
+        }
+        _ => Err(EvaluateError::FunctionRequiresMapValue(name).into()),
+    }
```
Comment (zmrdltl) [inline core/src/executor/evaluate/function.rs:777]: only one test left!
https://coveralls.io/builds/61509376/source?filename=core%2Fsrc%2Fexecutor%2Fevaluate%2Ffunction.rs#L775
```rust
test!(
        "SELECT UNWRAP('abc', 'a.b.c') FROM MapType",
    Err(EvaluateError::FunctionRequiresMapValue("UNWRAP".to_owned()).into())
    );
```

**Review by zmrdltl — CHANGES_REQUESTED** [review]

File: test-suite/src/function/entries.rs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+test_case!(entries, async move {
+    test! {
+        name: "test entries function works while creating a table simultaneously",
+        sql: "CREATE TABLE Item (map MAP)",
+        expected: Ok(Payload::Create)
+    };
+    test! {
+        name: "test if the sample string gets inserted to table",
+        sql: "INSERT INTO Item VALUES ('{\"name\":\"GlueSQL\"}')",
```
Comment (zmrdltl) [inline test-suite/src/function/entries.rs:14]: we can use r# operator to remove '\' for readable code
```rust
r#"INSERT INTO Item VALUES ('{"name":"GlueSQL"}')"#,
```

Comment (2-NOW) [reply test-suite/src/function/entries.rs]: thanks for the feedback. i've applied it :)

File: core/src/executor/evaluate/function.rs (hunk 1 above)
Comment (2-NOW) [reply core/src/executor/evaluate/function.rs:777]: thanks for the feedback. i've applied it :)

**Review by ever0de — APPROVED** [review]

**Review by panarch — APPROVED** [review]
Looks all nice! Thanks a lot for the contribution 👍 👍

File: core/src/ast/function.rs [hunk 3]
```diff
@@ -1176,14 +1178,6 @@ mod tests {
             ))))
             .to_sql()
         );
-
-        assert_eq!(
```
Comment (panarch) [inline core/src/ast/function.rs:1180]: this removal does not look intentional which is not related to this pr.

Comment (2-NOW) [reply core/src/ast/function.rs]: I fixed :)
>associated commit: 7d2eeb8

**Review by panarch — APPROVED** [review]
Looks all nice! Thanks a lot for the contribution 👍 👍

**Pull request merged**